consumer-services/src/main/scala/za/co/absa/spline/consumer/service/model/ExecutionPlanInfo.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.consumer.service.model
+
+import java.util.UUID
+
+import io.swagger.annotations.{ApiModel, ApiModelProperty}
+import za.co.absa.spline.consumer.service.model.ExecutionPlanInfo.Id
+
+@ApiModel(description = "Execution plan information")
+case class ExecutionPlanInfo
+(
+  @ApiModelProperty(value = "Execution plan Id")
+  _id: Id,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name '_id' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4579224)